package/tests/test_api/test_graphql.py

```diff
@@ -61,3 +242,72 @@ def test_graphql_runs_metadata_endpoint_no_dbsession(self, client):
                 json={"query": "{runMetadata(runIds: [1534326]) {id bookmark}}"},
             )
         assert response.json() == {"data": {"runMetadata": []}}
+
+    @patch("kedro_viz.api.graphql.get_run_tracking_data")
+    def test_graphql_runs_tracking_data_endpoint(
+        self, patched_tracking, client, example_tracking_output
+    ):
+        patched_tracking.return_value = example_tracking_output
+        response = client.post(
+            "/graphql",
+            json={
+                "query": """{runTrackingData(runIds: [1534326,1234,1234])
```

Comment:
This test is confusing. We request these 3 run ids here and in the response, each metric only has 1 run in it. And that run doesn't match any of the three given run ids.